package/docker/scripts/reports/sql/NoOfDiagnosis.sql

```diff
@@ -0,0 +1,6 @@
+SELECT "visit_diagnoses"."coded_diagnosis" AS "coded_diagnosis", count(*) AS "count"
```

Comment:
is there a specific reason to put the table and column names in quotes?

Reply:
Removed